website/src/theme/DocItem/Footer/contributors.jsx

```diff
@@ -0,0 +1,87 @@
+// biome-ignore lint/correctness/noUnusedImports: Needs this for React to work
+import React, { useState, useEffect } from 'react';
+import styles from './contributors.module.scss';
+
+const unleashTeam = new Map([
+    ['alvinometric', 'developer advocate, Unleash'],
+    ['andreas-unleash', 'senior software engineer, Unleash'],
+    ['chriswk', 'senior software engineer, Unleash'],
+    ['daveleek', 'senior software engineer, Unleash'],
+    ['FredrikOseberg', 'senior software engineer, Unleash'],
+    ['gardleopard', 'senior software engineer, Unleash'],
+    ['gastonfournier', 'senior software engineer, Unleash'],
+    ['ivarconr', 'co-founder, Unleash'],
+    ['kwasniew', 'senior software engineer, Unleash'],
+    ['nnennandukwe', 'developer advocate, Unleash'],
+    ['nunogois', 'senior software engineer, Unleash'],
+    ['sighphyre', 'senior software engineer, Unleash'],
+    ['sjaanus', 'senior software engineer, Unleash'],
+    ['thomasheartman', 'senior software engineer, Unleash'],
+    ['Tymek', 'senior software engineer, Unleash'],
+]);
+
+const GitHubContributors = ({ filePath }) => {
+    const [contributors, setContributors] = useState([]);
+
+    useEffect(() => {
+        const fetchFileContributors = () => {
+            fetch(
+                `https://api.github.com/repos/unleash/unleash/commits?path=${filePath}`,
+            )
+                .then((response) => response.json())
+                .then((commits) => {
+                    const contributorSet = new Set();
+                    // using a Set to deduplicate the list of contributors
+                    for (const commit of commits) {
+                        contributorSet.add(JSON.stringify(commit.author));
+                    }
+                    const contributors = Array.from(contributorSet).map((str) =>
+                        JSON.parse(str),
+                    );
+                    setContributors(contributors);
+                })
+                .catch((error) => {
+                    console.error(error);
+                    setContributors([]);
+                });
+        };
+
+        fetchFileContributors();
+    }, []);
+
+    if (!contributors.length) {
+        return <h3>Fetching contributors...</h3>;
+    }
+
+    return (
+        <div className={styles.contributors}>
+            <h3>Contributors</h3>
+
+            <div className={styles.wrapper}>
+                {contributors?.map((contributor) => {
+                    return (
+                        <figure className={styles.contributor}>
+                            <a href={contributor.html_url}>
+                                <img
+                                    src={contributor.avatar_url}
+                                    alt={contributor.login}
```

Comment:
I think this could imply that they work at GitHub, maybe we could do @alvinometric for example?

Reply:
I guess it could? Doesn't really read that way to me, but I'm not particularly bothered about this. My goal here is to make it clear to the end user where the link goes, but I think you can inspect the link for it, so I'm easy.